templates/sfdxPlugin/tsconfig.json

```diff
@@ -1,8 +1,9 @@
 {
   "extends": "./node_modules/@salesforce/dev-config/tsconfig",
   "compilerOptions": {
+    "declaration": true,
     "outDir": "./lib",
-    "rootDir": "./src"
+    "importHelpers": true
```

Comment:
This change fixes breakpoints being in the incorrect lines when stopping and running the VSCode debugger. It makes the debugging experience consistent when running tests and running the command in debug mode (e.g. --dev-suspend).
We do this by using the compiled code when running tests instead of the in-memory compilations that happen by compiling what's in `rootDir`.

Separate to this, we also start using `importHelpers` in order to reduce the size of the compiled code for those modules that use helper functions. More info on that here https://github.com/microsoft/tslib

Reply:
How confident are you with this change? I think oclif uses this property to determine if it should ts-node which is a huge developer improvement when developing plugins. https://github.com/oclif/config/blob/a3e3052c4acfece3acd66bab5a13ba7a32f7f6f9/src/ts-node.ts#L100

Reply:
The problem I ran into with the way oclif compiles during development vs once "released" was that the compilation was different. So debugging a production problem was difficult to replicate and debug because of that. With these changes prod and development are the same.
We've been developing and releasing with this configuration on both `sfdx-diff` and `salesforcedx-templates` plugins. So far it's been a good experience, we've been in production since end of August with the diff plugin and for 3 weeks with the templates one.